### DISCUSSION TITLE
po-popup não permite scroll

### DISCUSSION THREAD

**Discussion by leticiatomasio** [po-ui/po-angular#964] [Tire suas dúvidas]
Boa tarde. Manipulando o po-popup, notei que o componente não permite scroll dentro do p-popup-header-template. É isso mesmo ou existe alguma forma de contornar esse comportamento?

**Comment by jhonyeduardo** [comment]
Boa tarde @leticiatomasio 

Pode nos passar um exemplo no https://stackblitz.com/edit/po-ui, para tentarmos ajudar?
